docs/source/tutorial.md

```diff
@@ -0,0 +1,168 @@
+ * Add a custom command to a project.
+ * Use the `run` action to launch external processes.
+
+If `conda-project` is not yet installed and started Project, follow the [Installation Instructions](index).
+
+## Create a new project
+
+This section of the tutorial explores the `create` and `activate` subcommands of conda-project.
+
+1. Open a Command Prompt or terminal window.
+
+2. Initialize the project in a new directory:
+   ```shell
+   conda-project create -n learn-cp --directory cp-tutorial --platform osx-arm64,linux-64 -c defaults -c conda-forge \
+       python=3.10 notebook r-essentials r-plotly r-shiny
+   ```
+
+   At this point, **conda-project** will create a new directory, and write environment, lock, and project files to the new
+   directory. This example passes the `osx-arm64` and `linux-64` platform architectures. This has the effect of speeding up
+   the package resolution and lock file generation.  If the `--platform` argument is omitted, package resolution will be
+   calculated across **conda-project**'s default list of platform architectures.
+
+   ```{note}
+   This tutorial issues the `create` action for a new project.  As noted in the [User Guide](user_guide), it is also possible
+   to create a project from an existing environment file.
+
+   If using windows, change the `--platform` argument to point to the correct architecture (win-64).
+   ```
+
+3. Navigate the newly created project directory:
+   ```shell
+   cd cp-tutorial
+   ```
+
+4. Investigate the contents of the newly created environment file:
+   ```shell
+   $ cat environment.yml
+   channels:
+     - defaults
+     - conda-forge
+   dependencies:
+     - python=3.10
+     - notebook
+     - r-essentials
+     - r-plotly
+     - r-shiny
+   platforms:
+     - osx-arm64
+     - linux-64
+   ```
+
+   ```{note}
+   Refer to [conda-project create](user_guide) for a detailed list of options.
+   ```
+
+5. Investigate the contents of the newly created conda project file:
+   ```yaml
+   $ cat conda-project.yml
+   name: learn-cp
+     environments:
+   default:
+     - environment.yml
+   variables: {}
+   commands: {}
+   ```
+
+   Note that the `name` key is populated with the value supplied in the `create` action.
+
+   An upcoming section of the tutorial will involve adding a custom command to the `commands` key. It's also
+   possible to define variables using the `variables` key.  These key varlue pairs are loaded as overrides
+   to the inherited execution environment when a `run` action is issued.
+
+6. Initiate the `prepare` action on the project:
+   ```shell
+   $ conda-project prepare
+   ```
+
+   ```{note}
+   As noted in a [preceding section](user_guide), the `prepare` action will initiate conda install
+   for the active environment.
+   ```
+
+7. Activate the project's default environment by calling the `activate` action:
+   ```shell
+   $ conda-project activate
```

Comment:
The activte step would not be necessary since you use `conda project run jupyter notebook` later. If you want to keep the activate step it would be valid to run just `jupyter notebook` later from the activated shell.

Reply:
Ah, yep. Good call out. I was trying to include a demonstration of `conda project run` doing ad-hoc commands, but this case is a little silly.

Reply:
Removed. [6d9b5eb](https://github.com/conda-incubator/conda-project/pull/72/commits/6d9b5ebc347cce33a74037d53edf95acaf21ecd5)